app/src/components/Home/Note.vue

```diff
@@ -30,3 +12,23 @@
     },
   };
 </script>
+
+<template>
+  <article>
+    <textarea
+      @blur='addNote($event)'>
```

Comment:
If we only add note on blur, will it call the addNote function after we clickout of an existing note?  We can solve that problem easily with an editNote for existing.